spec/mentions_spec.rb

```diff
@@ -95,4 +95,21 @@ def para(content)
       expect(basic_post.output).to start_with(para(result.sub(default_src, mentions_src)))
     end
   end
+
+  context "when pulling the base from github metadata" do
+    let(:mentions_src) { "https://example.com" }
+    let(:config_overrides) do
+      {
+        "github" => { "mention_url_base" => mentions_src }
```

Comment:
Uh I think this is a completely different context so we don't have access to the GitHub metadata drop. We could, however, expose it via a Ruby API and if the GitHubMetadata gem exists, we could use that.